search-app/src/cmr/search/services/acls/collections_cache.clj

```diff
@@ -88,6 +90,7 @@
                                          (for [{:keys [provider-id EntryTitle] :as coll}
                                                collections]
                                            [[provider-id EntryTitle] coll]))]
+    (debug "fetch-collections-map count" (count collections))
     ;; We could reduce the amount of memory here if needed by only fetching the collections that
     ;; have granules.
     {:by-concept-id by-concept-id
```

Comment:
Should we write a ticket to reduce the amount of memory by only fetching the collections that have granules?  Do we need the collections in this cache that don't have granules?

Reply:
that comment was already there, I did not change the logic of what was going to be cached